incubator/hnc/internal/reconcilers/object_test.go

```diff
@@ -45,6 +45,18 @@ var _ = Describe("Secret", func() {
 		cleanupObjects(ctx)
 	})
 
+	PIt("should propagate objects only to selected namespace", func() {
```

Comment:
Comment why this is suppressed and when it should be unsuppressed

Reply:
Done